release.sh

```diff
@@ -135,8 +135,8 @@ if [[ ! $PUBLISH ]]; then
         fi
 
         # Edit docs to use CSS from CDN
-        grep -l '{@stylesheet: katex.min.css}' docs/*.md | xargs sed -i.bak \
-            's|{@stylesheet: katex.min.css}|<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@./dist/katex.min.css" integrity="sha384-katex.min.css" crossorigin="anonymous"/>|'
+        grep -l '/static/' docs/*.md | xargs sed -i.bak \
+            's|/static/|https://cdn.jsdelivr.net/npm/katex@./dist/katex.min.css" integrity="sha384-|'
```

Comment:
This seems a little fragile going forward. What if we used an image from `/static/`? Maybe we should use a regex (with `egrep` and `sed`) like `static/[^ ]*css` or `static/katex.min.css`. Oh, also, the replacement string is incorrect as is, but would be correct with such a query.

Reply:
@edemaine I've changed to only update URLs prefixed with `katex`, e.g., `/static/katex.min.css`.

Reply:
Isn't the second half of the `s` command still incorrect?

Reply:
@edemaine Sorry, I forgot the original replacement string was `{@stylesheet: katex.min.css}`. Other static assets than KaTeX and `index.css` are not served from `/static/`, so I think it's safe to assume it's built KaTeX files. I've changed to capture the file name.

Reply:
Did you forget to push, or are you still working on this?

Reply:
@edemaine I've amended the last commit. (4dbf4fc)